liblangutil/CharStreamBySourceName.h

```diff
@@ -0,0 +1,55 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+// SPDX-License-Identifier: GPL-3.0
+/**
+ * Interface to retrieve the scanner by a source name.
```

Comment:
```suggestion
 * Interface to retrieve the CharStream by a source name.
```

BTW, I see that the docstrings at the top of `CharStream.h`/`.cpp` also say "scanner".

Reply:
I think you missed this one.

And as I mentioned above, `CharStream.h`/`.cpp` have this:
```c++
/**
 * @author Christian <c@ethdev.com>
 * @date 2014
 * Solidity scanner.
 */
```